datashader/glyphs/points.py

```diff
@@ -1,4 +1,5 @@
 from __future__ import absolute_import, division
+from distutils.version import Version
```

Comment:
I think this needs to be packaging.version rather than distutils.version, because of [distutils now being deprecated ](https://www.python.org/dev/peps/pep-0632/) and slated for removal in an upcoming Python release (see https://github.com/holoviz/holoviews/pull/5180).

Reply:
thanks, fixed this